src/pytest_bdd/parser.py

```diff
@@ -165,7 +196,7 @@ def render(self, context: Mapping[str, Any]) -> Scenario:
             )
             for step in self._steps
```

Comment:
I wonder, are steps in the background supposed to be rendered with the example rows as well? Gherkin docs don't mention it though

Reply:
Very good question.  I'm not 100% either about that.

Reply:
let's leave it like that for now then